evident/q2/plugin_setup.py

```diff
@@ -150,14 +163,16 @@
 rm_param_descs["measurements"] = "Number of measurements per subject."
 rm_param_descs["correlation"] = "Correlation between repeated measurements."
 rm_param_descs["epsilon"] = "Sphericity parameter."
+rm_param_descs["data_column"] = "Column in metadata containing data."
 
 plugin.methods.register_function(
-    function=alpha_power_analysis_repeated_measures,
-    inputs={"alpha_diversity": SampleData[AlphaDiversity]},
-    input_descriptions={"alpha_diversity": "Alpha diversity vector"},
+    function=univariate_power_analysis_repeated_measures,
+    inputs={"data": SampleData[AlphaDiversity]},
```

Comment:
I don't think you need a specific input for this. As far as I recall you should be able to transform AlphaDiversity on-the-fly into a "metadata column". Or am I missing something here?

Reply:
That's odd, based on what I see on `q2-metadata` [inputs are not required for actions](https://github.com/qiime2/q2-metadata/blob/master/q2_metadata/plugin_setup.py#L32). If I understand this right, the way to set this up for evident is to remove the inputs, and leave just the metadata parameters so the Q2 machinery can transform and join the artifacts on-the-fly.

Reply:
Gotcha, gotcha. What I was doing was just removing the `input` lines completely. I think if we keep those lines but just pass empty dictionaries it should work. I'll try it out and update. Thanks!!!

Reply:
Excellent!